gravity.sh

```diff
@@ -937,6 +942,16 @@ Available options:
   esac
 }
 
+ReturnFileDescriptor() {
```

Comment:
```suggestion
return_file_descriptor() {
```

Keeping in the style of the rest of the functions. (Yes there are a few function names that are not styled correctly but the vast majority of function names are lowercase underbar space.